docs/release-notes/cloud/cloud-2024-04.md

```diff
@@ -0,0 +1,101 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud April 2024.
+included_jira_versions: ['2024.04']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/7.10.272
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/7.10.305
+---
+
+# Cloud April 2024
+
+These release notes are for the Codacy Cloud updates during April 2024.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+<!--TODO Check these issues manually
+
+-->
+
+## Product enhancements
+
+-   Improved [Security and risk management](../../organizations/managing-security-and-risk.md) with:
+    -   A redesigned overview page, offering comprehensive metrics and summary graphs for better visibility and tracking of security findings.
+    -   An improved findings table, which now includes a streamlined view of security findings. (TAROT-2546)
+    
+    !!! note
+        With this update, all references to "security items" will be updated to "security findings" to align with the new features.```
+
+    ![Security and risk management](../images/tarot-2546.png)
+
+-   You can now manually trigger a coverage pull request analysis using the [reanalyzeCoverage](https://api.codacy.com/api/api-docs#reanalyzecoverage) API endpoint. This can help you unblock pull requests that expect to receive a coverage report but only change uncovered files. (ALA-1003)
```

Comment:
Leaving one more tweak after clarifying some details on the [thread](https://codacy.slack.com/archives/C04EJUDAL8J/p1714750871711179?thread_ts=1714744232.731459&cid=C04EJUDAL8J). I'm also removing the use case, as it's useful only in edge cases and a detailed explanation is out of scope here IMO.

```suggestion
-   You can now manually trigger the reanalysis of the coverage of a pull request using the [reanalyzeCoverage](https://api.codacy.com/api/api-docs#reanalyzecoverage) API endpoint. (ALA-1003)
```